go.mod

```diff
@@ -10,43 +10,46 @@ require (
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/gophercloud/gophercloud v0.19.0
 	github.com/gophercloud/utils v0.0.0-20210720165645-8a3ad2ad9e70
-	github.com/onsi/gomega v1.17.0
-	github.com/openshift/api v0.0.0-20220322000322-9c4998a4d646
+	github.com/onsi/gomega v1.18.1
+	github.com/openshift/api v0.0.0-20220525145417-ee5b62754c68
 	github.com/openshift/build-machinery-go v0.0.0-20211213093930-7e33a7eb4ce3
-	github.com/openshift/client-go v0.0.0-20211209144617-7385dd6338e3
-	github.com/openshift/hypershift v0.0.0-20220323152148-c356b8b72d66
-	github.com/openshift/library-go v0.0.0-20211220195323-eca2c467c492
+	github.com/openshift/client-go v0.0.0-20220525160904-9e1acff93e4a
+	github.com/openshift/library-go v0.0.0-20220525173854-9b950a41acdc
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.11.0
-	github.com/spf13/cobra v1.2.1
+	github.com/prometheus/client_golang v1.12.1
+	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	github.com/vishvananda/netlink v1.1.0
-	golang.org/x/net v0.0.0-20211209124913-491a49abca63
+	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae // indirect
+	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.23.3
-	k8s.io/apiextensions-apiserver v0.23.0
-	k8s.io/apimachinery v0.23.3
-	k8s.io/client-go v0.23.3
-	k8s.io/code-generator v0.23.0
-	k8s.io/component-base v0.23.3
-	k8s.io/klog/v2 v2.30.0
-	k8s.io/kube-proxy v0.22.1
-	k8s.io/utils v0.0.0-20211208161948-7d6a63dca704
-	sigs.k8s.io/cluster-api-provider-openstack v0.0.0-a384cbe0dfa0c2543f6d35f024431a12494e670a
-	sigs.k8s.io/controller-runtime v0.11.0
+	k8s.io/api v0.24.0
+	k8s.io/apiextensions-apiserver v0.24.0
+	k8s.io/apimachinery v0.24.0
+	k8s.io/client-go v0.24.0
+	k8s.io/code-generator v0.24.0
+	k8s.io/component-base v0.24.0
+	k8s.io/klog/v2 v2.60.1
+	k8s.io/kube-proxy v0.24.0
+	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	sigs.k8s.io/cluster-api-provider-openstack v0.3.3
+	sigs.k8s.io/controller-runtime v0.12.0
 )
 
+require github.com/openshift/hypershift v0.0.0-20220525174911-c7c2b57c98ca
```

Comment:
This is weird that this ended up as a separate line... can you try moving it in with the rest of the `require`s and see if it still verifies?